File: in_go/src/pattern_21.go
package src

import (
	"fmt"
)

func Pattern_21() {
	row := 5
	column := 10
	symbol := "*"

	for i := 0; i < row; i++ {
		for j := 0; j < column; j++ {
			if j >= row-1-i && j <= column-1-i {
				fmt.Print(symbol)
			} else {
				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}

File: in_rust/src/pattern_21.rs
pub fn pattern_21() {
    let row = 5;
    let column = 10;
    let symbol = "*";
    for i in 0..row {
        for j in 0..column {
            if j >= row - 1 - i && j <= column - 1 - i {
                print!("{}", symbol)
            } else {
                print!(" ");
            }
        }
        println!();
    }
}
